lib/route.js

```diff
@@ -137,11 +137,19 @@ function buildRouting (options) {
         if (supportedMethods.indexOf(method) === -1) {
           throw new Error(`${method} method is not supported!`)
         }
+
+        if ((method === 'GET' || method === 'HEAD') && opts.schema && opts.schema.body) {
```

Comment:
I think we can extract this part of code as a function. It becomes large and duplicated.